### PR TITLE
fix: peer dependency duplicated with dependency

### PR DESCRIPTION
**Pull request by dead-horse**
This change is [Reviewable](https://reviewable.io/reviews/cnpm/npminstall/209)

### CODE REVIEW

File: test/peer.test.js [hunk 1]
```diff
@@ -30,4 +30,15 @@ describe('peer.test.js', () => {
...
   });
+
+  it('should ignore peerDependency if in dependencies', function* () {
+    yield coffee.fork(bin, [ 'react-countup@1.3.0' ], { cwd: tmp })
+      .debug()
+      .expect('code', 0)
+      .end();
+    assert(getPkg('node_modules/react-countup/node_modules/react/package.json').version.startsWith('15.'));
+    const pkg = getPkg('node_modules/react-countup//package.json');
+    assert(pkg.dependencies.react === '^15.3.2');
```
Comment (dead-horse) [inline test/peer.test.js]: 奇葩模块

Comment (fengmk2) [inline test/peer.test.js]: `//` 多了一个斜杠

File: lib/install.js [hunk 2]
```diff
@@ -226,14 +226,21 @@ function* _install(parentDir, pkg, ancestors, options) {
...
+    const pkgMaps = deps.prodMap;
 
     const peerDependencies = realPkg.peerDependencies || {};
     if (Object.keys(peerDependencies).length) {
       const unmatched = {};
       for (const name in peerDependencies) {
+        const version = peerDependencies[name];
+        // don't need to check if peer dependency is in dependencies
+        if (pkgMaps[name]) {
+          unmatched[name] = version;
```
Comment (fengmk2) [inline lib/install.js]: 不需要提示了

**Comment by dead-horse** [comment]
去掉了，先合并发布

**Comment by fengmk2** [comment]
+1

**Pull request merged**